src/rtc.rs

```diff
@@ -0,0 +1,484 @@
+/*!  Interface to the real time clock */
```

Comment:
Please use the single-line comment style for all comments, i.e. `//! ...` in this case